core/src/main/kotlin/pages/PageBuilder.kt

```diff
@@ -103,5 +134,5 @@ interface PageBuilder {
     fun pageForModule(m: Module): ModulePageNode
     fun pageForPackage(p: Package): PackagePageNode
     fun pageForMember(m: CallableNode): MemberPageNode
-    fun pageForClass(c: Class): ClassPageNode
+    fun pageForClass(c: ClassLike): ClassPageNode
```

Comment:
1) if it takes a ClassLike as a param, then it's probably not a `pageForClass`
2) if it's not a `pageForClass` then it shouldn't return a `ClassPageNode`

Reply:
Renamed.